app/controllers/task_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Task controller
+class TaskController < ApplicationController
```

Comment:
Naming convention for controller is plural, see this gist [0] for naming conventions in ruby and rails

https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e#controllers